cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/connector/ConnectorSpec.java

```diff
@@ -31,10 +31,22 @@ public class ConnectorSpec {
   // schema is null when the connector is unable to retrieve it from the resource
   private final Schema schema;
   private final Set<PluginSpec> relatedPlugins;
+  private final Set<String> supportedSampleTypes;
 
-  private ConnectorSpec(@Nullable Schema schema, Set<PluginSpec> relatedPlugins) {
+  private ConnectorSpec(@Nullable Schema schema,
+                        Set<PluginSpec> relatedPlugins,
+                        Set<String> supportedSampleTypes) {
     this.schema = schema;
     this.relatedPlugins = relatedPlugins;
+    if (supportedSampleTypes != null) {
+      this.supportedSampleTypes = supportedSampleTypes;
+    } else {
+      this.supportedSampleTypes = new HashSet<>();
+    }
+  }
+
+  private ConnectorSpec(@Nullable Schema schema, Set<PluginSpec> relatedPlugins) {
```

Comment:
Is this used at all? I see the only constructor call from the Builder and it's migrated to 3-parameter constructor

Reply:
I don't believe so, as I cannot find any references outside of the Builder either. I have removed this.